site2/tools/docker-build-site.sh

```diff
@@ -27,7 +27,7 @@ ROOT_DIR=$(git rev-parse --show-toplevel)
 cd $ROOT_DIR/pulsar-client-cpp
 
 BUILD_IMAGE_NAME="${BUILD_IMAGE_NAME:-apachepulsar/pulsar-build}"
-BUILD_IMAGE_VERSION="${BUILD_IMAGE_VERSION:-ubuntu-16.04}"
+BUILD_IMAGE_VERSION="${BUILD_IMAGE_VERSION:-ubuntu-16.04-py2}"
```

Comment:
where and how do we build this image ?
apachepulsar/pulsar-build:ubuntu-16.04-py2

Reply:
The image was uploaded manually by @merlimat before. The dockerfile is `build/docker/Dockerfile`.